src/components/SingleReview.jsx

```diff
@@ -18,6 +18,25 @@ const Review = () => {
         })
     }, [review_id])
 
+    const voteOnReview = () => {
+
+        setReview((review) => {
+            return {...review, votes: review.votes + 1}
+            
+        })
+        const reqBody = {
+            inc_votes: 1,
+        };
+        axios.patch(`https://be-games-example-api.herokuapp.com/api/reviews/${review.review_id}`, reqBody).then(({data}) => {

```

Comment:
See comment on line 21 regarding the ${review.review_id} you are passing in here.